Updates to the contributing process (#141)
EVERSE-ResearchSoftware/RSQKit: CONTRIBUTING.md

   one of the other methods described below.

2. If you have a [GitHub][github] account, or are willing to [create
   one][github-join], but do not know how to use Git, you can report problems
   or suggest improvements by [creating an issue][issues]. This allows us to
   assign the item to someone and to respond to it in a threaded discussion.

3. If you are comfortable with GitHub, and would like to add or change material,
   you can submit a pull request (PR). Instructions for doing this are
   [included below](#creating-a-pull-request).

### What to Contribute

At this moment, we welcome:

- [reports on issues, bugs and typos](#reporting-an-issue) in RSQKit content
- suggestions for new content by creating a [new issue](#reporting-an-issue) for discussion
- comments on [existing issues][issues] and reviews of [existing pull requests][pull-requests]
- [pull requests](#creating-a-pull-request) to fix issues, bug and typos in the RSQKit content

If you plan to submit a pull request that adds new content or significantly changes existing content, 
please [contact the Editorial Board][contact] to discuss this first so that 

> the effort is not spent by making a change that may not be accepted.

Reviewer: ```suggestion
the effort is only spent on making a change that will be be accepted.
```
